store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/serialisation/ElementSerialisation.java

```diff
@@ -42,13 +43,20 @@
 import java.nio.charset.Charset;
 import java.util.Arrays;
 import java.util.Iterator;
-import java.util.Map;
 
 public class ElementSerialisation {
     private static final Logger LOGGER = LoggerFactory.getLogger(ElementSerialisation.class);
 
     private final Schema schema;
 
+    protected enum EdgeDirection {
```

Comment:
It feels like we have declarations for variations on this "EdgeDirection" enum in a few different places - would it be better to have one implementation in the core module?